### PR TITLE
WebContents: emit did-fail-load event when url exceeds character limit

### PR DESCRIPTION
**Pull request by deepak1556**
Refs https://github.com/electron/electron/issues/8448

### CODE REVIEW

File: spec/api-browser-window-spec.js [hunk 1]
```diff
@@ -218,6 +218,17 @@ describe('BrowserWindow module', function () {
...
     })
 
+    it('should emit did-fail-load event for URL exceeding character limit', function (done) {
+      w.webContents.on('did-fail-load', function (event, code, desc, url, isMainFrame) {
+        assert.equal(desc, 'ERR_INVALID_URL')
+        assert.equal(code, -300)
+        assert.equal(isMainFrame, true)
+        done()
+      })
+      let data = new Buffer(2 * 1024 * 1024).toString('base64')
```
Comment (kevinsawicki) [inline spec/api-browser-window-spec.js]: Looks like this could be `const`

File: spec/api-browser-window-spec.js [hunk 2]
```diff
@@ -218,6 +218,17 @@ describe('BrowserWindow module', function () {
...
+      w.loadURL('data:image/png;base64,' + data)
```
Comment (kevinsawicki) [inline spec/api-browser-window-spec.js]: What do you think about using a template string here?

```js
`data:image/png;base64,${data}`
```

**Comment by deepak1556** [comment]
Have made the changes, thanks!

**Comment by kevinsawicki** [comment]
Thanks for fixing this @deepak1556 👍

**Pull request merged**